versions/wmde12.env

```diff
@@ -1,34 +1,34 @@
 WMDE_RELEASE_VERSION=wmde.12
```

Comment:
The changes to the env file look good to me
Mostly copied from https://github.com/wmde/wikibase-release-pipeline/blob/wmde.10/versions/wmde10.env and then altered as far as I can see

Reply:
yes, that's exactly what I did